### PR TITLE
Talk mo-runiverse-posit 

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

File: content/events/2023-09-19-mo-runiverse-posit.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+date: 2023-09-19T19:30:00 # UTC!! same as dateEnd
+description: "At Posit Conf? Come to learn from Mo about R-Universe!"
+location: 'Chicago'
+country: '🇺🇸' # emoji
+attendees:
+  - Athanasia Mo Mowinckel
+resources: # can be added later, one entry per talk (don't add while still empty, add once there are resources)
+
+---
+
```
Comment (yabellini) [inline content/events/2023-09-19-mo-runiverse-posit.md:14]:
```suggestion

At Posit Conf? Come to learn from Mo about R-Universe!

```

**Comment by yabellini** [comment]
@drmowinckels would you like to review this event we create for your talk about r-universe?

File: content/events/2023-09-19-mo-runiverse-posit.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+At Posit Conf? Come to learn from Mo about R-Universe!
+
+Say goodbye to installation headaches and hello to a universe of possibilities with R-Universe! Take your R package development to new frontiers by organizing and sharing packages beyond the bounds of CRAN. R-Universe's reliable package-building process strengthens installation and usage instructions, resulting in fewer support requests and an easy installation experience for users. With webpages and an API for exploring packages, R-Universe creates a streamlined and tidy ecosystem for R-package constellations. Also, you can build a custom toolchain for your users, relieving your workload and empowering users to help themselves. Join me to learn how to explore the vastness of R-Universe and expand your package development possibilities!
```
Comment (drmowinckels) [inline content/events/2023-09-19-mo-runiverse-posit.md:17]:
```suggestion
Say goodbye to installation headaches and hello to a universe of possibilities with R-Universe! Take your R package development to new frontiers by organizing and sharing packages beyond the bounds of CRAN. R-Universe's reliable package-building process strengthens installation and usage instructions, resulting in fewer support requests and an easy installation experience for users. With webpages and an API for exploring packages, R-Universe creates a streamlined and tidy ecosystem for R-package constellations. Join me to learn how to explore the vastness of R-Universe and expand your package development possibilities!
```

**Review by drmowinckels — CHANGES_REQUESTED** [review]
Thank you for this!

**Pull request merged**